slu-scripts/src/index.ts

```diff
@@ -59,6 +68,17 @@ export async function execScript(argv: any) {
 			console.error(ex.message);
 			process.exit(1);
 		}
+	} else if (argv._.includes('send-data')) {
+		try {
+			if (interval) {
+				setInterval(sendData, Number(interval), encryptedDataPath, keyFilePath, isConfiguration);
```

Comment:
You have missed to add the payload here. Maybe it is better to write is as following so you are aware of missing parameters. 

````
setInterval( () => sendDate(params), Number(interval))
```

Also applies for the other interval